stack/tasks/release.js

```diff
@@ -165,60 +199,109 @@ const confirmedRelease = (type, version) => {
     return;
   }
 
-  waterfall([
-    (callback) => {
-      exec(`git checkout ${developTrunk} && git pull && git checkout -b release-tmp`, handleSuccess(callback, () => {
-        console.log(chalk.cyan(outdent`
+  let releaseSteps;
+
+  if (type === HOTFIX) {
```

Comment:
i would externalise common funcs and reuse them

Reply:
you `isHotfix`? Thougt so too